maxArrSize/Home 03, 04 -Doc Manager/src/main/java/service/UserService.java

```diff
@@ -0,0 +1,40 @@
+package service;
+
+import model.User;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserService {
+    private List<User> usersList = new ArrayList<>();
+
+    public UserService() {
+     User admin = new User("admin", "admin");
+    }
+
+    public void addUser(User user){
+        usersList.add(user);
+    }
+
+    public void deleteUser(User user){
+        if (checkRegisteredUsers(user)){
+            usersList.remove(user);
+        }
+    }
+
+    public boolean checkRegisteredUsers(User user) {
+        boolean isUserExists = false;
+        if ((user.getUserName() != null) && (user.getUserPass() != null)) {
```

Comment:
Strange method. Better write equals method for User class, and check usersList.contains(user);